### PR TITLE
Profile card for chaise

### PR DESCRIPTION
**Pull request by nitishksahu**
- A dropdown is added to the profile name that's displayed on the navbar.

- Upon clicking the profile name, a profile card appears with all the other information found in 
   webauthn's client object.

- The card has all the group names, the user is a part of, which is clickable and will navigate to 
   globus group detail page when clicked.

### CODE REVIEW

**Comment by jrchudy** [comment]
Where are the tests for this feature?

File: common/templates/navbar.html [hunk 1]
```diff
@@ -40,7 +49,10 @@
...
                     </li>
                 </ul>
-            </div><!--/.nav-collapse -->
+            </div><!--/.nav-collapse -->    
         </div>
     </nav>
 </header>
+
+<!--Modal Window -->
+<div profilemodal id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true" class="modal fade" ></div>
```
Comment (jrchudy) [inline common/templates/navbar.html]: Why does this modal element need to be defined here? 

In all other cases we use the modal popup, the modal is a template that is rendered into the DOM with only having to specify it as part of the `$uibModal.open` function.

Comment (jrchudy) [reply common/templates/navbar.html]: In my local code, removing this element from the DOM doesn't appear to change any of the functionality.

**Comment by nitishksahu** [comment]
@jrchudy the requested changes have been done and the PR is now available for review.

**Comment by nitishksahu** [comment]
The changes have been merged to master, assigning this to @hongsudt to verify and close.

**Pull request merged**